butterknife-runtime/src/test/java/butterknife/BindViewTest.java

```diff
@@ -3,14 +3,22 @@
 import butterknife.compiler.ButterKnifeProcessor;
 import com.google.common.collect.ImmutableList;
 import com.google.testing.compile.JavaFileObjects;
+
+import javax.lang.model.element.Element;
 import javax.tools.JavaFileObject;
 import javax.tools.StandardLocation;
+
 import org.junit.Test;
 
+import static butterknife.TestStubs.ANDROIDX_CONTEXT_COMPAT;
+import java.util.List;
+import java.util.Map;
+
```

Comment:
not sure imports order is right, and no need to add empty lines between them

Reply:
What do you think of applying the spotless plugin in ButterKnife ? I can file a ticket for that.

Reply:
Jake can decide, I don't know. My point is rather that whatever rule was applied before is the correct way to organize imports.